ONSdigital/ras-frontstage
# Fix password reset with email change

# What and why?
Adds error handling for when a password reset token has been generated but the email has been changed since (invalidating the token and deleting it from the party database) and the user tries to use the old reset token

# How to test?
- deploy to environment
- follow the forgot password path for an account
- change the user email and verify the change through the link provided
- try using the password reset link (should show the `Your link is invalid or has already been used` page)
# Jira
[RAS-840](https://jira.ons.gov.uk/browse/RAS-840)
